content/spin/v2/http-outbound.md

```diff
@@ -102,9 +102,9 @@ const response = await fetch("https://example.com/users");
 
 **Notes**
 
-* Although the underlying Spin interface is blocking, the `fetch` function is defined by JavaScript as async. You must await the response, but the request will always block, and the promise will resolve as soon as the request is returned.
+You can find a complete example of using outbound HTTP in the JavaScript SDK repository on [GitHub](https://github.com/fermyon/spin-js-sdk/tree/main/examples/common-patterns/outbound-http)
```

Comment:
The `spin.ts` only wraps the user provided handler and attaches it to the FetchEvent.

Reply:
Oh!  So the "true" entry point is the `handleEvent` function, but that's unfriendly, so we shunt it off into this "pay no attention to the man behind the curtain" file, which hands off to the friendly entry point?  And we can't do this inside the SDK because it needs to import the user handler?

So this is what the stuff about "the SDK attaches `handle` to the FetchEvent" thing was - "the SDK" here is this user-side bit rather than the `@fermyon/spin-sdk` library!  This all makes way more sense now!

This is non-blocking but I think

1. we should explain this, at least in passing, in (a relevant part of) the docs
2. we should add a comment to the template file about it being required plumbing with which they should not mess

Or maybe this is all just obvious to JS folks, it just wasn't to me...!